File: good_pair.py
# A pair (i, j) is called good if nums[i] == nums[j] and i < j.
# output : total numbers of pairs in the array.

def fun(x, nums):
    count = 0
    for i in range(0, x):
        if nums[i] == nums[x]:
            count += 1
    return count

def good_pair(nums):
    n = len(nums)
    ans = 0
    for x in range(0, n):
        ans += fun(x, nums)
    return ans

print(good_pair([1, 2, 1, 1, 3, 4]))

File: goodPair.java
public class goodPair {
    public static void main(String[] args) {
        int[] nums = {1, 2, 3, 1, 1, 3};
        System.out.println(goodPairs(nums));
    }
    public static int goodPairs(int[] nums) {
        int count = 0;
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] == nums[j]) {
                    count++;
                }
            }
        }
        return count;
    }

    
}